src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,13 @@ public User get(String login) {
         }
         return null;
     }
+
+    @Override
+    public boolean clearStorage() {
+        Storage.people.clear();
+        if (Storage.people.size() == 0) {
```

Comment:
Equation `Storage.people.size() == 0` already returns the boolean.

You can simplify your code by removing redundant if statements, e.g. 
`return Storage.people.size() == 0`